core/src/test/java/org/kohsuke/stapler/interceptor/JsonOutputFilterTest.java

```diff
@@ -117,16 +117,12 @@ public MyData getSomeIncludedData() {
     public void doIndex(StaplerResponse rsp) throws IOException {
         rsp.setContentType("text/html");
         PrintWriter w = rsp.getWriter();
-        w.println("<html><body><script src='prototype'></script><script src='script'></script>");
+        w.println("<html><body><script src='script'></script>");
         w.println("<script>var v = "+ WebApp.getCurrent().boundObjectTable.bind(this).getProxyScript()+";</script>");
-        w.println("<script>var callback = function(t){var x=t.responseObject(); alert(Object.toJSON(x)); };</script>");
+        w.println("<script>var callback = function(t){var x=t.responseObject(); alert(JSON.stringify(x)); };</script>");
```

Comment:
Since this test suite doesn't have Prototype present, it's OK to call `JSON.stringify()` here. (The test suite in core, and core itself in production, does have Prototype present, though.)